docs/sources/installation/helm/monitor-and-alert/index.md

```diff
@@ -27,8 +26,63 @@ Rules and alerts are automatically deployed.
 - Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
 - A running Kubernetes cluster with a running Loki deployment.
 - A running Grafana instance.
-- A running Prometheus operator in order for recording rules for the dashboards
-  to work.
+- A running Prometheus Operator installed using the `kube-prometheus-stack` Helm chart.
+
+**Prometheus Operator Prequisites**
+
+The included dashboards require certain metric relabelings be applied to Kubernetes metrics. The best way to accomplish this is to install the `kube-prometheus-stack` Helm chart with the following values file, replacing `CLUSTER_NAME` with the name of your cluster. The cluster name is what you specify during the helm installation, so a cluster installed with the command `helm install loki-cluster grafana/loki` would be called `loki-cluster`.
+
+```yaml
+kubelet:
+  serviceMonitor:
+    cAdvisorRelabelings:
+      - action: replace
+        replacement: <CLUSTER_NAME>
+        targetLabel: cluster
+      - targetLabel: metrics_path
+        sourceLabels:
+          - "__metrics_path__"
+      - targetLabel: "instance"
+        sourceLabels:
+          - "node"
+
+defaultRules:
+  additionalRuleLabels:
+    cluster: <CLUSTER_NAME>
+
+"kube-state-metrics":
+  prometheus:
+    monitor:
+      relabelings:
+        - action: replace
+          replacement: <CLUSTER_NAME>
+          targetLabel: cluster
+        - targetLabel: "instance"
+          sourceLabels:
+            - "__meta_kubernetes_pod_node_name"
+
+"prometheus-node-exporter":
+  prometheus:
+    monitor:
+      relabelings:
+        - action: replace
+          replacement: <CLUSTER_NAME>
+          targetLabel: cluster
+        - targetLabel: "instance"
+          sourceLabels:
+            - "__meta_kubernetes_pod_node_name"
+
+prometheus:
+  monitor:
+    relabelings:
+      - action: replace
+        replacement: <CLUSTER_NAME>
+        targetLabel: cluster
+```
+
+The `kube-prometheus-stack` installs `ServicMonitor` and `PrometheusRule` resources for monitoring Kubernetes, and it depends on the `kube-state-metrics` and `prometheus-node-exporter` helm charts which also install `ServiceMonitor` resources for collecting `kubelet` and `node-exporter` metrics. The above values file adds the necessary additional relabelings are required for these metrics to work with the included dashboards.
+
+If using this helm chart in an environment which does not allow for the installation of `kube-prometheus-stack` or custom CRDs, it is recommended you run `helm template` on the `kube-prometheus-stack` helm chart with the above values file, and grab all generated `ServiceMonitor` and `PrometheusRule` resources. Note those resources may have to be modified with the correct ports and selectors to find the various services such as `kubelet` and `node-exporter` in your environment.
```

Comment:
```suggestion
If you are using this helm chart in an environment which does not allow for the installation of `kube-prometheus-stack` or custom CRDs, you should run `helm template` on the `kube-prometheus-stack` helm chart with the above values file, and grab all generated `ServiceMonitor` and `PrometheusRule` resources. These resources may have to be modified with the correct ports and selectors to find the various services such as `kubelet` and `node-exporter` in your environment.
```
And I want another word for "grab", but I'm not sure what's appropriate here.  "note"? "collect"?

Reply:
how about "review"?

Reply:
That works.